apps/webservice/src/app/[workspaceSlug]/(app)/(deploy)/(raw)/systems/[systemSlug]/(raw)/deployments/[deploymentSlug]/(sidebar)/_components/release-cell/ReleaseEnvironmentCell.tsx

```
@@ -15,28 +15,28 @@ import {
+import { DeploymentVersionStatus } from "@ctrlplane/validators/releases";
 
 import { useDeploymentVersionChannelDrawer } from "~/app/[workspaceSlug]/(app)/_components/channel/drawer/useDeploymentVersionChannelDrawer";
 import { ApprovalDialog } from "~/app/[workspaceSlug]/(app)/(deploy)/_components/release/ApprovalDialog";
 import { ReleaseDropdownMenu } from "~/app/[workspaceSlug]/(app)/(deploy)/_components/release/ReleaseDropdownMenu";
 import { api } from "~/trpc/react";
 import { DeployButton } from "./DeployButton";
-import { Release } from "./TableCells";
+import { DeploymentVersion } from "./TableCells";
 
-type Release = {
+type DeploymentVersion = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'DeploymentVersion'. Consider to delete it or rename it.

'DeploymentVersion' is defined here:


(lint/suspicious/noRedeclare)